### PR TITLE
Add installation instructions for pip

### PR DESCRIPTION
**Pull request by heitorPB**
Several issues report problems with using `pip install pyproj`. A workaround is using the latest version from Github.

Some issues with pip problems: #141 #136 #133 #132 #130 #59 #25 #13 #9.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,16 @@ Examples of how to set the PROJ_DIR environment variable:
...
 * Linux/OS X on most shells- `$ export PROJ_DIR=/lib/`
 
+An alternative way to install is with `pip`:
+
+```
+pip install cython
+pip install git+https://github.com/jswhit/pyproj.git
+```
+
+You migh want to run pip with administrative privilegies (e.g. `sudo pip`) or
```
Comment (milieu) [inline README.md]:
```suggestion
You might want to run pip with administrative privileges (e.g. `sudo pip`) or
```

Comment (milieu) [reply README.md]:
```suggestion
You may need to run pip with administrative privilegies (e.g. `sudo pip`) or
```

File: README.md [hunk 2]
```diff
@@ -17,6 +17,16 @@ Examples of how to set the PROJ_DIR environment variable:
...
+perform an user only installation (e.g. `pip install --user`).
```
Comment (milieu) [inline README.md]:
```suggestion
perform a user only installation (e.g. `pip install --user`).
```
I know, I know, [dumb exceptions](https://english.stackexchange.com/a/105117) in human language 😜

Comment (heitorPB) [reply README.md]: OMG, didn't know about this. Thanks!

File: README.md (hunk 1 above)
Comment (heitorPB) [reply README.md]: Fixed!

**Pull request merged**